### ISSUE TITLE
mysql: using a unix domain socket

### ISSUE THREAD

**Issue by philips** [volatiletech/sqlboiler#425]
### What version of SQLBoiler are you using (`sqlboiler --version`)?

SQLBoiler v3.1.0

How do I use a unix domain socket. For example the a DSN with the Go mysql driver might be:

`root@unix(/var/run/mysqld/mysqld.sock)/public`

**Comment by philips** [comment]
Never mind, seems hard coded to TCP https://github.com/volatiletech/sqlboiler/blob/master/drivers/sqlboiler-mysql/driver/mysql.go#L121

**Comment by philips** [comment]
Is there a particular reason you don't just use DSNs instead of YAML/TOML in this project?

**Comment by aarondl** [comment]
Yeah, the reason being is because we pick that apart to be able to create the database in the tests. At that point we need it broken apart and didn't want to end up parsing DSNs.
